src/surge-xt/gui/widgets/Switch.cpp

```diff
@@ -59,9 +59,10 @@ void Switch::mouseDown(const juce::MouseEvent &event)
         return;
     }
 
+    notifyControlModifierClicked(event.mods);
+
```

Comment:
We need to guard this with the event type we want to send. We can't send it unprotected for all mouse downs. Same in MultiSwitch